docs/live-tests-guide.md

```diff
@@ -0,0 +1,214 @@
+# Live Tests Guide
+
+This guide explains how to run live tests and generate recordings for supported providers (VLLM and Vertex AI).
+
+## Supported Providers
+
+- **vllm** - VLLM inference server (default, requires local container)
+- **vertex** - Google Cloud Vertex AI
+
+The script auto-detects the provider from environment variables. If no provider is specified, it defaults to vllm.
+
+## Local Testing
+
+### Prerequisites
+
+1. **Podman** - Required for building and running containers (including vllm container)
+2. **gcloud CLI** - For GCP authentication (if using Vertex AI provider)
+3. **Git** - For cloning repositories
```

Comment:
These documents should assume basic things like `Git` or `Python` are installed.